discord_thread_summarizer_using_openai/ai_utils.bal

```diff
@@ -0,0 +1,23 @@
+import ballerinax/openai.chat;
+
+configurable string openAIToken = ?;
+
+chat:Client chatClient = check new ({
+    auth: {
+        token: openAIToken
+    }
+});
+
+function generateChatCompletion(string prompt, string model = "gpt-3.5-turbo") returns string|error {
+    chat:CreateChatCompletionRequest req = {
+        model: model,
+        messages: [{"role": "user", "content": prompt}]
+    };
+    chat:CreateChatCompletionResponse res = check chatClient->/chat/completions.post(req);
+    string? content = res.choices[0]?.message["content"];
+    if (content == ()) {
+        return error("The message is empty.");
+    } else {
+        return content;
+    }
```

Comment:
```suggestion
    if content is string {
        return content;
    }
    
    return error("The message is empty.");
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/1dbf76e067fd08cc0430182aa02b7c98d154f3ac